telemetryui/telemetryui/templates/record_details.html

```diff
@@ -40,6 +40,15 @@
     <dt>Architecture</dt>
     <dd>{{ record.architecture }}</dd>
 
+    <dt>Board Name</dt>
+    <dd>{{ record.board_name }}</dd>
```

Comment:
Do you think we should substitute a string (e.g. `N/A`) if the default value is set, or else it would appear empty? We do this for `guilty_id` at present...

Reply:
I agree a `<N/A>` for consistency  should be used.